docs/HowTo/Use/node_network_migration.md

```diff
@@ -0,0 +1,107 @@
+---
+description: Migrate node to a new IP network
+---
+
+# Migrate node to a new IP network
+
+When changing the network configuration of a node, it's recommended to follow specific steps so the new configuration it's applied and the consensus protocol continues to work as expected.
+
+!!!important
+
+    Before starting the migration, create a backup of your data directory of all Nodes.
+
+## Prerequisites
+
+- [GoQuorum installed](../GetStarted/Install.md)
+- A running [Raft network](../../Tutorials/Create-a-Raft-network.md) or [IBFT network](../../Tutorials/Create-IBFT-Network.md)
+
+## Raft
+
+### Peers with same networking configuration
+
+This applies when the node has a new networking configuration but the peers keep the same IP/Ports as before.
+
+For example:
```

Comment:
Examples are a bit tricky to understand because it is not clear who is node and who is peers.

Maybe you do something like 

```
For example assuming a 3 nodes network A, B, C

- Node A changes public IP/Ports and B & C keep the same public IP/Ports
...
```

Reply:
yes, doing that. I actually used in the first version the `Node X` as the migrated node and peers the others, then removed. 

Do you think `Node X` will be fine or should I use A/B/C?

Reply:
I think being explicit will have benefits to really understand the big picture.

Reply:
updating

Reply:
I think using A/B/C will be clearer

Reply:
done